src/storage/allocator.ts

```diff
@@ -272,7 +289,9 @@ export function getAllocationOperationFromField(
                     size,
                 };
             } else if (src.format !== undefined && src.format !== null) {
-                throw Error("Unsupported struct format " + src.format);
+                throwInternalCompilerError(
```

Comment:
ESLint has a false positive for this case.

`throwInternalCompilerError` returns `never`, but it treats it like it doesn't terminate control flow here. If we replace this line with `throw new Error("42")`, there won't be a warning.

In other cases, it doesn't show false positives. I assume the problem is in their logic of handling control flow termination in if-else clauses.

So, I'm suppressing this.

---

Let me know if you need any further adjustments!